Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -1315,16 +1315,19 @@ std::pair<int, int> HGCalDDDConstants::waferTypeRotation(
     int layer, int waferU, int waferV, bool fromFile, bool debug) const {
   int type(HGCalTypes::WaferOut), rotn(0);
   int wl = HGCalWaferIndex::waferIndex(layer, waferU, waferV);
+  bool endOfList(true);
   if (fromFile && (waferFileSize() > 0)) {
     auto itr = hgpar_->waferInfoMap_.find(wl);
-    if (itr != hgpar_->waferInfoMap_.end()) {
+    endOfList = (itr != hgpar_->waferInfoMap_.end());
```

Comment:
Am I reading this line correctly? `endOfList` is true if the end of the list has NOT been reached? Isn't that backwards? Am I missing something?